getting_started.rst

```diff
@@ -137,47 +269,27 @@ command line or taken from the defaults in *<userhome>/.conan/profiles/default*
    :width: 500 px
    :align: center
 
-For example, the command :command:`conan install . -s os="Linux" -s compiler="gcc"`, performs these steps:
+For example, the command :command:`conan install . --settings os="Linux" --settings compiler="gcc"`, performs these steps:
 
 - Checks if the package recipe (for ``Poco/1.9.0@pocoproject/stable`` package) exists in the local cache. If we are just starting, the
   cache is empty.
 - Looks for the package recipe in the defined remotes. Conan comes with `conan-center`_ Bintray remote as the default, but can be changed.
 - If the recipe exists, the Conan client fetches and stores it in your local cache.
-- With the package recipe and the input settings (Linux, GCC), the Conan client will validate that the corresponding binary is in the local
-  cache. This test will not run when installing for the first time.
-- The Conan client searches for the corresponding binary package in the remote. It will be fetched if it exists.
-- The Conan client will then  generate the requested files specified in the ``[generators]`` section.
-
-The Conan client will throw an error If the binary package required for specific settings doesn't exist. It is possible to try to build the
-binary package from sources using the :command:`--build=missing` command line argument to install. A detailed description on how to build a
-binary package from sources is described in the below sections.
-
-.. warning::
-
-    In the Bintray repositories there are binaries for several mainstream compilers and versions, such as Visual Studio 12, 14, Linux GCC
-    4.9 and Apple Clang 3.5. If you are using a different setup, running the command might fail because of the missing package. You could
-    try to change your settings or build the package from source, using the :command:`--build=missing` option, instead of retrieving the
-    binaries. Such a build might not have been tested and may eventually fail.
+- With the package recipe and the input settings (Linux, GCC), Conan looks for the corresponding binary is in the local cache.
+- Then Conan searches the corresponding binary package in the remote and fetches it.
+- Finally, it generates an appropriate file for the build system specified the ``[generators]`` section.
 
-Building the Timer Example
---------------------------
+There are binaries for several mainstream compilers and versions available in Conan Center repository in Bintray, such as Visual Studio 14,
+15, Linux GCC 4.9 and Apple Clang 3.5... Conan will throw an error if the binary package required for specific settings doesn't exist. You
+can try to build the binary package from sources using :command:` conan install --build=missing`. You will find more info
```

Comment:
```suggestion
can build the binary package from sources using :command:` conan install --build=missing`, it will succeed if your configuration is supported by the recipe. You will find more info
```